code/backend/batch/utilities/helpers/AzureSearchHelper.py

```diff
@@ -24,7 +58,10 @@ def search_dimensions(self) -> int:
             )
         return AzureSearchHelper._search_dimension
 
-    def get_vector_store(self):
+    def get_client(self):
+        return self.search_client
+
+    def create_or_update_index(self):
```

Comment:
This is named Create or Update but I don't think it updates?

Reply:
I'll verify this and rename accordingly when/if we do this for real